emp/emp_role_cftemplate.yaml

```diff
@@ -0,0 +1,370 @@
+              # below.
+              - ec2:DescribeRouteTables
+              - ec2:DeleteRoute
+              - ec2:CreateRoute
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              # pricing data is mostly public, it's needed for cost calculations
+              - pricing:GetProducts
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              - iam:GetRole
+              - iam:GetRolePolicy
+              - iam:ListAttachedRolePolicies
+              - iam:ListRolePolicies
+              - iam:GetPolicy
+              - iam:GetPolicyVersion
+              - iam:SimulateCustomPolicy
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              - iam:PassRole
+            Resource:
+              - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${EKSRole}'
+            Effect: Allow
+          - Action:
+              # these are api calls our controllers make, we should be able to restrict these once we have tagging.
+              # the tagging permissions are left as-is for that purpose
+              - elasticfilesystem:CreateFileSystem
+              - elasticfilesystem:CreateMountTarget
+              - elasticfilesystem:CreateTags
+              - elasticfilesystem:DeleteFileSystem
+              - elasticfilesystem:DeleteMountTarget
+              - elasticfilesystem:DeleteTags
+              - elasticfilesystem:DescribeFileSystems
+              - elasticfilesystem:DescribeMountTargets
+              - elasticfilesystem:DescribeTags
+              - elasticfilesystem:ListTagsForResource
+              - elasticfilesystem:TagResource
+              - elasticfilesystem:UntagResource
+            Resource:
+              - '*'
+            Effect: Allow
+          - Action:
+              # required by baremetalpool controller check ConfigureSGRules
+              - ec2:AuthorizeSecurityGroupEgress
+              # needed by kubevirt to migrate ebs volumes attached to pods, resource here is * because we can't dictate the
+              # attributes of ebs attached. the customer has full freedom to restrict access to limited resources, and we
+              # encourage to do it according to their use case.
+              - ec2:AttachVolume
+              - ec2:DetachVolume
+              - ec2:DescribeVolumes
+              # needed by emp-server to populate the data on the UI. We can't dictate what resources customer will use therefore resource *
+              # the customer has full freedom to restrict access to limited resources, and we encourage to do it according to their use case.
+              - ec2:DescribeRouteTables
+              - ec2:DescribeVpcs
+              - ec2:DescribeRegions
+              - ec2:DescribeSecurityGroups
+              - ec2:DescribeSubnets
+              - ec2:DescribeKeyPairs
+              - ec2:DescribeAvailabilityZones
```

Comment:
Most of them taken care in CAPA permissions

Reply:
see: https://github.com/platform9/support-locker/pull/50#discussion_r1743087737
putting it here is intentional. They'll be removed once we have tagging in place and know for sure the exact scope of each permission